TreeDB/cmd/treedb_rag_benchmark/application_runner.go

```
@@ -0,0 +1,1928 @@
+	if len(cfg.Command) == 0 {
+		cfg.Command = append([]string(nil), os.Args...)
+	}
+	return applicationProvenance{
+		ProductBaseSHA: cfg.ProductBaseSHA, HarnessRevision: revision, BinarySHA256: binaryHash,
+		FixtureSHA256: applicationFixtureDigest(fixture), ConfigSHA256: applicationConfigDigest(cfg),
+		SemanticVectorSHA256: bundle.Digest(), HashingRegressionSHA256: hex.EncodeToString(hashingSum[:]),
+		GoVersion: runtime.Version(), GOOS: runtime.GOOS, GOARCH: runtime.GOARCH, CGOEnabled: cgo,
+		Hostname: hostname, HostNote: cfg.HostNote, Command: cfg.Command,
+		RepetitionOrder: "query reps forward/reverse/forward; ingestion fresh DB reps 0..4; final candidate must use paired ABBA interleave",
+		Environment: map[string]string{
+			"GOROOT": os.Getenv("GOROOT"), "runtime_goroot": runtime.GOROOT(), "GOMAXPROCS": fmt.Sprint(runtime.GOMAXPROCS(0)),
```

golangci-lint (2.12.2)

[error] SA1019: runtime.GOROOT has been deprecated since Go 1.24: The root used during the Go build will not be meaningful if the binary is copied to another machine. Use the system path to locate the “go” binary, and use “go env GOROOT” to find its GOROOT. 

(staticcheck)